Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get -qq update && apt-get install --no-install-recommends -y apt-transpo
     pip install --upgrade pip && \
     apt-get purge -y software-properties-common && \
     apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-RUN git clone --depth 1 --single-branch --branch rel_17.05_plus_k8s_fs_support_resource_limit_reqs https://github.com/phnmnl/galaxy.git
+RUN git clone --depth 1 --single-branch --branch phnmnl-integration/isa https://github.com/phnmnl/galaxy.git
```

Comment:
Can we change the name to something like `17.05_fs_supp-res_lim_req-isa`? That requires renaming the branch at the other side of course. This is mostly to know what things I need to cherry pick when moving galaxy versions (to 17.09) soon.

Reply:
Ok, I'll make a new branch on galaxy repos, and change that clone command.

Reply:
We already have a branch!

Reply:
Yes, I would just rename the branch, but either way, rename or create a new from the last commit of `phnmnl-integration/isa`, is fine for me at least. I just want to have a reminder of what we have on top of the galaxy release that we are starting from (and which one is that galaxy release), as the galaxy git tree is a bit painful to the eye to look :-).

Reply:
https://github.com/phnmnl/galaxy/commits/feature/isa-datatype

Reply:
Sorry, which one? I said to Marco that I was taking it from there some time ago, since I need to synchronize with mtbls-dwnld new version. If you give me the name of the branch, I will make a merge and see if there are any differences. I've already merge two recent modifications by Marco inside the branch.

Reply:
You're right Luca, that was the branch. But we need another branch for integration within more recent PhenoMeNal Galaxy version. The branch feature/isa-datatype does not contain other PhenoMeNal modifications and thus could be proposed as a pull request to the Galaxy project.